atomics/T1220/T1220.yaml

```diff
@@ -18,38 +19,45 @@ atomic_tests:
       description: Location of the test XSL script file on the local filesystem.
       type: Path
       default: PathToAtomicsFolder\T1220\src\msxslscript.xsl
+    msxsl_exe:
+      description: Location of the MSXSL executable.
+      type: Path
+      default: PathToAtomicsFolder\T1220\bin\msxsl.exe
```

Comment:
Would the expected behavior to be for someone to download msxsl.exe to PathToAtomicsFolder\T1220\bin\msxsl.exe since it doesn't exist by default in this directory?

Reply:
Disregard. I see how you handle this in the get-prereq command. I like it!